docs/contributing/introduction-to-contributing.md

```diff
@@ -151,4 +151,4 @@ Do you have questions? Join the conversation in our [Discord](https://discord.gg
 ## License
 
 By contributing to the OpenSauced project, you agree that your contributions will be licensed
```

Comment:
[Proofreading] It seemed a bit repetitive seeing `license` and `licensed` so I provided another alternative. 
```suggestion
By contributing to the OpenSauced project, you agree that your contributions will be authorized
```

Reply:
I don't think we can do this as authorized and licensed don't have the same meaning in this context.

Reply:
Hey @dominicduffin1. I changed my suggestion. Take a look.

Reply:
I don't think this language works because the license isn't to permit or authorize the contribution but rather the contributor is agreeing to grant a license on their contributions to the repo.